generators/entity/index.js

```diff
@@ -1091,6 +1091,9 @@ class EntityGenerator extends BaseBlueprintGenerator {
                 if (this.isJhipsterVersionLessThan('5.0.0')) {
                     this.removeFile(`${constants.ANGULAR_DIR}entities/${entityName}/${entityName}.model.ts`);
                 }
+                if (this.isJhipsterVersionLessThan('6.3.0')) {
+                    this.removeFile(`${constants.ANGULAR_DIR}entities/${context.entityFolderName}/index.ts`);
```

Comment:
can you please also add a client framework check here?

Reply:
I am merging this as I need this before next upgrade

Reply:
Change done. Thanks for pointing.